### DISCUSSION TITLE
[Question] Confused on expo-auth-session deprecation on Facebook and Google

### DISCUSSION THREAD

**Discussion by hyochan** [expo/expo#23835] [Archived: Questions]
After reviewing the documentation at https://docs.expo.dev/versions/latest/sdk/auth-session/#facebook, I've come to understand that expo-auth-session is phasing out support for Facebook and Google. I further confirmed this through deprecated notices in pr #23126.

If we proceed to migrate the Facebook SDK as outlined in https://docs.expo.dev/guides/facebook-authentication, it appears that we would no longer be able to handle Facebook authentication on the web. This was one of the key features that made `expo-auth-session` valuable. The same issue seems to apply to Google as well.

I want to ensure that I'm correctly understanding this situation. If I'm missing or misinterpreting anything, could you please clarify?

**Comment by stesvis** [comment]
That is my understanding too. I am very disappointed because I was just able now to make it work!
